server/routes/incidents_routes.py

```
@@ -1595,14 +1595,45 @@ def apply_fix_suggestion(user_id, suggestion_id: str):
-        from chat.backend.agent.tools.github_apply_fix_tool import github_apply_fix
-
-        result_json = github_apply_fix(
-            suggestion_id=suggestion_id_int,
-            use_edited_content=use_edited_content,
-            target_branch=target_branch,
-            user_id=user_id,
-        )
+        # Determine which provider owns this suggestion's repository
+        provider = "github"  # default fallback
+        with db_pool.get_admin_connection() as conn:
+            with conn.cursor() as cursor:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)